doc/protocol.md

```diff
@@ -1072,6 +1072,13 @@ The following options can be specified in the "open" control message:
    you don't set this field, the actual tag will not be checked.  To
    express that you expect the file to not exist, use "-" as the tag.
 
+* "attrs": a JSON object containing file owner and group information
+           to set. Supporting the file ownership fields returned by `fsinfo`.
+ * `uid`: an integer, the uid of the file owner (`st_uid`)
+ * `owner`: a string, or an integer, the uid of the file owner (`st_uid`)
+ * `gid`: an integer, the gid of the file group (`st_gid`)
+ * `group`: a string, or an integer, the gid of the file group (`st_gid`)
```

Comment:
Can you please use `-` here or indent this a little more, to make this easier to read?